### PR TITLE
Fixed broken URL in help output

### PR DESCRIPTION
**Pull request by jlstevens**
This PR addresses #978. The code change is trivial but it is likely that the test data will need updating.

### CODE REVIEW

**Comment by jlstevens** [comment]
I'm a little surprised that the tests passed...maybe this output is being sent to the pager and therefore isn't captured?

**Comment by philippjfr** [comment]
Yes, I think that's it. One issue I have is that custom Elements will also point to non-existent entries in the elements tutorial. Got any ideas to disable that?

**Comment by jlstevens** [comment]
I agree that is an issue though not a particularly major one. My only suggestion is to suggest the help URL only if the element is one of the ones we offer. Do we have such a list?

Maybe the top level ``__init__.py`` could set a global variable (list of all our elements) in ``core.pprint``. At least this would avoid having to pass this list though help, then Store, then into the pretty printer and all its methods...

**Comment by philippjfr** [comment]
Could probably just iterate over the ``hv.element`` namespace and find all ``Element`` types to get that list.

**Comment by jlstevens** [comment]
I suppose so given that seaborn elements aren't in that tutorial anyway. Should I really do the same for Containers or can I just ignore the possibility of custom Containers? I doubt people will want to build new containers often...

**Comment by philippjfr** [comment]
>Should I really do the same for Containers or can I just ignore the possibility of custom Containers?

Fine to ignore right now I think.

**Comment by jlstevens** [comment]
Added filtering of elements as suggested:

image

Versus a core element:

image

Note this PR isn't quite ready to merge. I might as well fix the issue about showing help for the active backend while I am working on this code.

File: holoviews/__init__.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
 from .operation import ElementOperation, MapOperation, TreeOperation # noqa (API import)
 from .element import *                                               # noqa (API import)
 from . import util # noqa (API import)
+from .element import __all__ as elements_list
```
Comment (philippjfr) [inline holoviews/__init__.py]: That doesn't seem right, it should at least filter for objects of type Element.

Comment (philippjfr) [inline holoviews/__init__.py]: I know ``hv.element.__all__`` filters for ViewableElement but I think that includes some other types as well.

Comment (jlstevens) [inline holoviews/__init__.py]: The problem is that ``__all__`` contains strings (which I want) but doing tighter subclass checks requires a list of those classes. I could iterate over ``locals`` but I find that pretty ugly. What would you suggest?

Comment (philippjfr) [inline holoviews/__init__.py]: I'd change line 99 of ``hv.element.__init__`` to:

``return issubclass(obj, Element)``

Comment (jlstevens) [inline holoviews/__init__.py]: Done! I was hesitant in case anything was expecting to find ViewableElements...

Comment (philippjfr) [inline holoviews/__init__.py]: Appears you were right being hesitant, but it really shouldn't be relying on importing Overlay via ``hv.element``.

**Comment by jlstevens** [comment]
Ok. If the tests pass I think this PR is ready for final review/merge.

**Comment by philippjfr** [comment]
Looks good, merging.

**Pull request merged**